addons/xinference-cluster/Chart.yaml

```diff
@@ -3,7 +3,7 @@ annotations:
 apiVersion: v2
 name: xinference-cluster
 type: application
-version: 0.7.0-alpha.0
```

Comment:
The chart version should be changed to [0.8.0-alpha.5](https://github.com/apecloud/helm-charts/releases/tag/xinference-0.8.0-alpha.5)